packages/block-editor/src/components/block-list/style.scss

```diff
@@ -109,8 +109,9 @@
 			border: $border-width solid transparent;
 			border-left: none;
 			box-shadow: none;
-			transition: border-color 0.1s linear, box-shadow 0.1s linear;
 			pointer-events: none;
+			transition: border-color 0.1s linear, box-shadow 0.1s linear;
+			@include reduce-motion;
```

Comment:
So how does SASS compile this? Is the `@media query` always before or after the original rule? 
Maybe it's not consistent enough to count on but I'm wondering if `!important` is even necessary if the `@media` comes after...

Reply:
Well theoretically the compiled order shouldn't matter, right? Since the added media query adds another level of specificity that would usually override the existing rule. 

With that in mind, I the `!important` only becomes necessary if someone happened to add a competing `transition-duration` property for the same element, later on in the stylesheet and more specific than this one. Probably rare?

Reply:
> Well theoretically the compiled order shouldn't matter, right? Since the added media query adds another level of specificity that would usually override the existing rule.

That's what I thought too until a couple days ago. According to the spec, media queries don't add any weight. 
Maybe some browsers don't follow that cause I could swear I've seen otherwise. :confused: 

I don't think `!important` is that out of line here. Might as well leave it.

Reply:
I tried removing it in 5350c8e, and haven't found any instances where these rules are ignored yet. 🤷‍♂️

Seeing as that's the case, I think we might as well keep it out until we determine we definitely need it.